projects/ion/src/lib/popover/component/popover.component.ts

```diff
@@ -14,6 +14,7 @@ export class IonPopoverComponent {
   @Input() ionPopoverBody: TemplateRef<void>;
   @Input() ionPopoverActions?: IonButtonProps[];
   @Input() ionPopoverIcon?: IconType;
+  @Input() ionPopoverIconColor? = '#0858ce';
```

Comment:
```suggestion
  @Input() ionPopoverIconColor? = PRIMARY_6;
```
I would add a name to this value.